### PR TITLE
Details popup window

### PR DESCRIPTION
**Pull request by abeltsew**
Dear Reviewer,

In this PR, The popup Detail window is implemented with javascript. Details are below.

- [ ] refactor cards section to be displayed dynamically
- [ ] store all information in a javascript object
- [ ] close popup on click of close button

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,9 +1,302 @@
...
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    shortDesc:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    liveLink: 'https://abeltsew.github.io/portfolio/',
+    sourceCode: 'https://github.com/abeltsew',
+  },
+];
+
+const popUpDetail = (project) => {
+  // const project = projects[parseInt(projectId) - 1];
```
Comment (tufoinnkuo10) [inline index.js]: - [x] kindly remove commented code to maintain clean code practice

File: index.js [hunk 2]
```diff
@@ -1,9 +1,302 @@
...
+    tech: ['html', 'Bootstrap', 'Ruby on rails'],
+    image: 'images/Detail_img.png',
+    description:
+      "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releaLorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releax map lapora verita.",
+    shortDesc:
+      "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard",
+    liveLink: 'https://abeltsew.github.io/portfolio/',
+    sourceCode: 'https://github.com/abeltsew',
+  },
+  {
```
Comment (tufoinnkuo10) [inline index.js:38]: - [x] great job. please your mobile version should follow the figma design. a user should not scroll before seeing the full popup. kindly reduce the height.
- [x] kindly make the tech buttons smaller as well like in the figma design
- [x] please make the close icon more visible in the mobile popup
- [x] kindly ensure that the `see live` and `see source` buttons are aligned with the text and image on the left and right side in the mobile popup.
yours
Screenshot (867)_LI
Screenshot (868)_LI
figma
Screenshot (866)

File: index.js [hunk 3]
```diff
@@ -1,9 +1,302 @@
...
-mobileMenu.addEventListener("click", () => toggleMenu());
\ No newline at end of file
+menu.addEventListener('click', () => toggleMenu());
+mobileMenu.addEventListener('click', () => toggleMenu());
+
+// Toogle pop Up Menu
+
+const projects = [
+  {
+    id: 1,
```
Comment (tufoinnkuo10) [inline index.js:15]: - [x] nice job. Please make your background a little bit darker/blurer when the desktop popup comes up(especially on the white sections) so the start and end of the popup can be seen. 
 - [x] please reduce the size of the tech icons on the popup so the match the figma design
yours
Screenshot (870)_LI
figma
Screenshot (871)

**Review by tufoinnkuo10 — CHANGES_REQUESTED** [review]
Hi @Team

There are some issues that you still need to work on to go to the next project but you are almost there!

To Highlight
✅ You used the correct Github flow
✅ Descriptive pull request message
✅ linter error checks validated
✅ Documented your work professionally
✅ popups ok and responsive
✅ information from projects array

Required Changes ♻️
Check the comments under the review
tufoin

Optional suggestions
_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by BregornOriginal — APPROVED** [review]
## Hi @abeltsew 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 

## Congratulations! 

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Good work making the changes requested by the previous reviewer

## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Pull request merged**